circle.yml

```diff
@@ -0,0 +1,11 @@
+machine:
+  node:
+    version: 0.10.34
+
+test:
+  override:
```

Comment:
isn't all this encapsulated in a check script, so you can just do `npm run check`?